### ISSUE TITLE
Update GitHub site workflow to share link on changed example file

### ISSUE THREAD

**Issue by stoyanK7** [checkstyle/checkstyle#13131]
From https://github.com/checkstyle/checkstyle/pull/13112#issuecomment-1571229085
> please create issue on website action to share Links to updated parts of it :)

**Comment by stoyanK7** [comment]
@romani I assume you meant that when an `Example.java` file is changed, there should be a link pointing to the `Examples` section, correct?

**Comment by nrmancuso** [comment]
@romani please clarify requirements

**Comment by romani** [comment]
Yes, at least link to Check page, detalization to specific examples might be too much

**Comment by romani** [comment]
this is already implemented, all changed checks are referenced in comment from Github action 
example: https://github.com/checkstyle/checkstyle/pull/17383#issuecomment-3068010227
